lib/screens/onboarding/create_account.dart

```diff
@@ -1,9 +1,11 @@
 import 'dart:async';
```

Comment:
Need to create another text field - Full Name - and then suggest account name based on it

Reply:
Maybe will implement this in another pull request

Reply:
will add this in the next PR.